shared/common-adapters/input.native.js

```diff
@@ -116,10 +116,15 @@ class Input extends Component<void, Props, State> {
     if (this.props.onEnterKeyDown && e.key === 'Enter') {
       this.props.onEnterKeyDown(e)
     }
+
+    if (this.props.onEscapeKeyDown && e.key === 'Escape') {
+      this.props.onEscapeKeyDown(e)
```

Comment:
i kinda think we shouldn't actually add this method. your thing can just pass in onKeyDown and look for escape instead of making this a thing all inputs know about